src/components/visualizations/utils/drilldownEventing.ts

```diff
@@ -65,13 +66,18 @@ function composeDrillContextGroup(
     points: IHighchartsPointObject[],
     chartType: ChartType,
 ): IDrillEventContextGroup {
+    const isCombo = isComboChart(chartType);
     const contextPoints: IDrillPoint[] = points.map((point: IHighchartsPointObject) => {
+        const customProps: Partial<IDrillPoint> = isCombo ? { type: get(point, "series.type") } : {};
```

Comment:
Please explain to me why it's not possible to return just `chartType`

Reply:
<img width="304" alt="drill on attribute" src="https://user-images.githubusercontent.com/39481271/59997764-04195b00-9688-11e9-8fcb-7cdce3eeecfd.png">
Because of the expectation in [SD-469](https://jira.intgdc.com/browse/SD-469). When drilling on an attribute in combo chart, should return chart type of each point (column/line/area). That helps user to know which chart type of each measure

Reply:
thanks for explanation